### PR TITLE
[Onboarding] refactor intents type to be an enum

### PR DESCRIPTION
**Pull request by xtina-starr**
Switch the `intents` input field in the `updateCollectorProfile` mutation to be an enum.

### CODE REVIEW

File: schema/me/update_collector_profile.js [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+export const IntentsType = new GraphQLEnumType({
+  name: "Intents",
+  values: {
+    BUY_ART_AND_DESIGN: {
+      value: "buy art & design",
+    },
+    SELL_ART_AND_DESIGN: {
+      value: "sell art & design",
+    },
+    RESEARCH_ART_PIECES: {
```
Comment (mzikherman) [inline schema/me/update_collector_profile.js]: This says `PIECES`, but the `value` has `prices`. Intentional?

Comment (xtina-starr) [reply schema/me/update_collector_profile.js]: lol not at all!

**Comment by alloy** [comment]
👍

File: schema/me/update_collector_profile.js [hunk 2]
```diff
@@ -16,7 +40,7 @@ export default mutationWithClientMutationId({
       type: GraphQLString,
     },
     intents: {
-      type: new GraphQLList(GraphQLString),
+      type: new GraphQLList(IntentsType),
     },
```
Comment (alloy) [inline schema/me/update_collector_profile.js]: Btw, is intenta required?

Comment (xtina-starr) [reply schema/me/update_collector_profile.js]: a required input field? no

**Comment by orta** [comment]
yep - good to go

**Pull request merged**